docs/upgrade/UPGRADE-7.0.0-beta3.md

```diff
@@ -0,0 +1,259 @@
+# [Upgrade from 7.0.0-beta2 to v7.0.0-beta3](https://github.com/shopsys/shopsys/compare/v7.0.0-beta2...v7.0.0-beta3)
+
+This guide contain instructions to upgrade from version 7.0.0-beta2 to 7.0.0-beta3.
+Before you start, don't forget to take a look at [general instructions](/UPGRADE.md) about upgrading.
+There you can find links to upgrade notes for other versions too.
+
+## [shopsys/coding-standards]
```

Comment:
shopsys/framewrk should be first, shouldn't it?

Reply:
Indeed. I moved it and update instructions about writing upgrade.